roles/nginx/templates/wordpress.conf

```diff
@@ -1,31 +1,48 @@
+# WordPress single site rules.
+# https://www.nginx.com/resources/wiki/start/topics/recipes/wordpress/
+# Designed to be included in any server {} block.
+
+# Upstream to abstract backend connection(s) for php
+upstream php {
+        server unix:/var/run/php-fpm/wordpress.sock;
+        server 127.0.0.1:9000;
+}
+
 server {
-    listen       443 default_server;
-    server_name  fossrit.jwf.io;
-    root         /var/www/wordpress/;
+        server_name     grantgreen.justinwflory.com;
+        root            /var/www/html;
+        index           index.html index.php;
+
+        location = /favicon.ico {
+            log_not_found   off;
+            access_log      off;
+        }
 
-	client_max_body_size 64M;
- 
-	# Deny access to any files with a .php extension in the uploads directory
-    location ~* /(?:uploads|files)/.*\.php$ {
-        deny all;
-    }
+        location = /robots.txt {
+            allow           all;
+            log_not_found   off;
+            access_log      off;
+        }
 
-    location / {
-        index     index.php index.html index.htm;
-        try_files $uri $uri/ /index.php?$args;
-    }
+        location / {
+                # This is cool because no php is touched for static content.
+                # include the "?$args" part so non-default permalinks don't break with query string
+                try_files $uri $uri/ /index.php?$args;
+        }
 
-    location ~* \.(gif|jpg|jpeg|png|css|js)$ {
-        expires max;
-    }
+        location ~ \.php$ {
+                #NOTE: You should have "cgi.fix_pathinfo = 0;" in php.ini
+                include                  fastcgi_params;
+                try_files                $uri =404;
+                fastcgi_split_path_info  ^(.+\.php)(/.+)$;
+                fastcgi_index            index.php;
+                fastcgi_param            SCRIPT_FILENAME $document_root$fastcgi_script_name;
+                fastcgi_intercept_errors on;
+                fastcgi_pass php;
+        }
 
-    location ~ \.php$ {
-        try_files               $uri =404;
-        fastcgi_split_path_info ^(.+\.php)(/.+)$;
-        fastcgi_index           index.php;
-        fastcgi_pass            unix:/var/run/php-fpm/wordpress.sock;
-        fastcgi_param           SCRIPT_FILENAME
-                                $document_root$fastcgi_script_name;
-        include                 fastcgi_params;
-    }
+        location ~* \.(js|css|png|jpg|jpeg|gif|ico)$ {
+                expires         max;
```

Comment:
@jwflory there's another part here if you want it

Reply:
@ct-martin Are you suggesting to remove this section? As I understand, this is instructing clients to not set an expiration in a local cache for any of these types of files, since they are usually static.